src/ui/components/Card/index.js

```diff
@@ -63,7 +63,7 @@ export default class Card extends React.Component {
           'Card--no-footer': !footer,
         })}
       >
-        {header ? <header className="Card-header">{header}</header> : null}
+        {header ? <h2 className="Card-header">{header}</h2> : null}
```

Comment:
It's not too clear to me why `header` is become a heading here since presumably `header` is being used as a container?

Reply:
I fixed this but the build is failing, even though there are no linting issues

Reply:
I again converted into a header tag with the tags. But the build is failing

Reply:
@muffinresearch I think the idea of the OP is to mark these card "headers" as headings because in most cases, we inject textual content in the `header` prop (which is often the title of that card).

Reply:
If this goes in the direction of changing this to a heading element, we'll need to check that there isn't any props passing complete headings anywhere.